src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -2893,8 +2901,9 @@ copy_bo_with_kdma(const std::shared_ptr<xrt_core::device>& core_device,
 
   // Construct a kernel command to copy bo.  Kernel commands
   // must be shared ptrs
+  xrt::hw_context hwctx;
```

Comment:
kernel_command should have another constructor without hwctx, i don't think creating a temp object to create kernel_command is good idea.

Reply:
kernel_command constructor declared as explicit. So, added hsctx as extra arg. do you want me to remove explicit and add new constructor with hw_ctx arg?

Reply:
sure